Release 1.10.4 - reduce error alerts, use GitHub Actions

briskt opened the pull request:
### Changed
- Don't send an alert for an error thrown by `/site/status` which is used by the ECS health check that has a very short retry time. It sends a flood of messages when there is a database issue, for instance.
- Switch from Codeship to GitHub Actions

briskt committed:
don't send an alert for error thrown by /site/status (ECS health check)

briskt committed:
add GitHub Actions workflow for test and publish

briskt committed:
change image name to use secrets.DOCKER_ORG

briskt committed:
use DOCKERHUB_USERNAME and DOCKERHUB_TOKEN per login-action README

briskt committed:
use a specified version of ubuntu for the Action runner

briskt committed:
remove Codeship config

briskt committed:
add actions-services.yml for GitHub Actions

briskt committed:
move DOCKER_ORG from secrets to vars

briskt committed:
move the Actions config to the top level of the repo

briskt committed:
use 503 Service Unavailable for /site/status (not 502 Bad Gateway)

briskt committed:
add missing comma

briskt committed:
Merge pull request #52 from silinternational/feature/limit-alerts

don't send an alert for error thrown by /site/status (ECS health check)

briskt committed:
Merge pull request #53 from silinternational/feature/actions

add GitHub Actions workflow for test and publish

briskt committed:
use ubuntu-latest GitHub runner

briskt committed:
suggestions from feature-branch review

jason-jackson commented on .github/workflows/test-and-publish.yml:
```diff
@@ -0,0 +1,58 @@
+  push:
+
+jobs:
+  unit-tests:
```
it might be nice to merge the tests into one job, but this works for now.

briskt replied:
I had hoped that running them in parallel would make it finish more quickly, but since the image build has to happen for each, it didn't help much. There's probably a way to optimize this by building the image first, then running the tests in parallel, but that's a task for another day.

briskt committed:
run tests in series to avoid building the image three times

 -- I had hoped that running these in parallel would save time, but it didn't work out that way.
